.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Magento 2 code
+
+Contributions to the Magento 2 codebase are done using the fork & pull model.
+This contribution model has contributors maintaining their own fork of the Magento 2 repository.
+The forked repository is then used to submit a request to the base repository to “pull” a set of changes.
+For more information on pull requests please refer to [GitHub Help](https://help.github.com/articles/about-pull-requests/).
+
+Contributions can take the form of new components or features, changes to existing features, tests, documentation (such as developer guides, user guides, examples, or specifications), bug fixes or optimizations.
+
+The Magento 2 development team or community maintainers will review all issues and contributions submitted by the community of developers in the first in, first out order.
+During the review we might require clarifications from the contributor.
+If there is no response from the contributor within two weeks, the pull request will be closed.
+
+For more detailed information on contribution please read our [beginners guide](https://github.com/magento/magento2/wiki/Getting-Started).
+
+## Contribution requirements
```

Comment:
We should change these contribution requirements, since we're not enforcing quite a few of the standards required in Magento core.

Some notes:
1. Docs go in this repo, rather than devdocs (want them versioned together with code)
2. We don't require following Magento coding standards

Reply:
Contribution requirements updated.